### ISSUE TITLE
"About your city" section of the City Profile page

### ISSUE THREAD

**Issue by CloudNiner** [azavea/temperate#502]
## Overview

Implement the "About your city" section of the city profile page. User should be able to see Compact of Mayors completion and edit the fields of this section.

## Screenshots
![screen shot 2018-02-12 at 5 28 06 pm](https://user-images.githubusercontent.com/5672295/36123278-1d23bbb8-101a-11e8-9233-20edebb3805e.png)

![screen shot 2018-02-12 at 5 30 01 pm](https://user-images.githubusercontent.com/5672295/36123366-645b5b12-101a-11e8-98ff-668178995571.png)

## Notes
- ngx-bootstrap has an accordion that can likely be used for this task
- Consider adding city profile fields either directly to the organization model, or to a new one-to-one city profile model
- Consider adding the accordion and backend model in a separate PR first, to unblock implementation of other UI sections
- A stubbed, empty page for the city profile already exists as `CityProfileComponent`

## Done criteria
- [ ] Only one open at a time accordion, with the "About your city" section implemented
- [ ] Fields should automatically save to the backend when a field loses focus
- [ ] Compact of Mayors count should update automatically and be reflected immediately in UI
- [ ] Tooltip for red badges describing "For Global Covenant of Mayors participants"
- [ ] Compact of Mayors count should update on in the dashboard city profile summary component
